py/dml/serialize.py

```diff
@@ -62,6 +68,21 @@ def declare_variable(site, name, type, init_expr = None):
     return (ctree.mkDeclaration(site, name, type, init = init),
             ctree.mkLocalVariable(site, symtab.LocalSymbol(name, name, type)))
 
+def prepare_array_de_serialization(site, t):
```

Comment:
nitpick: "de_serialization" is incorrect

Reply:
This was actually intentional, meant to reflect the slash in `de/serialization` -- but yeah, it's questionable. I'd welcome a better suggestion.